packages/databricks-vscode/src/configuration/auth/DatabricksCliCheck.ts

```diff
@@ -82,22 +92,24 @@ export class DatabricksCliCheck implements Disposable {
         );
 
         try {
-            await workspaceClient.currentUser.me();
+            await workspaceClient.currentUser.me(
```

Comment:
Just for my understanding: the workspace client will throw some "cancelled" error if the token is cancelled, right? (so we will hit `return false` case)

Reply:
yep